utils/test_helper.py

```diff
@@ -0,0 +1,293 @@
+
+def create_test_normal_user(num=1):
+    for i in range(num):
+        username = "test_user%s" % (i)
+        password = "test_user%s" % (i)
+        user_level = User.USER
+        create_test_user(username, password, user_level)
+
+def get_test_admin_user(index=0):
+    name = "test_admin%s" % (index)
+    return User.objects.get(username=name)
+
+def get_test_judge_user(index=0):
+    name = "test_judge%s" % (index)
+    return User.objects.get(username=name)
+
+def get_test_normal_user(index=0):
+    name = "test_user%s" % (index)
+    return User.objects.get(username=name)
+
+def get_test_admin_client(index=0):
+    test_admin_client = Client()
+    text = "test_admin%s" % (index)
+    test_admin_client.login(username=text, password=text)
+    return test_admin_client
+
+def get_test_judge_client(index=0):
+    test_judge_client = Client()
+    text = "test_judge%s" % (index)
+    test_judge_client.login(username=text, password=text)
+    return test_judge_client
+
+def get_test_normal_user_client(index=0):
+    test_user_client = Client()
+    text = "test_user%s" % (index)
+    test_user_client.login(username=text, password=text)
+    return test_user_client
+
+def create_problem(owner, pname=None, visible=False):
+    if pname == None:
+        pname = random_word(20)
+    problem = Problem.objects.create(pname=pname, owner=owner, visible=visible)
+    return problem
+
+def random_word(length):
+    return ''.join(random.choice(string.lowercase) for i in range(length)) 
+
+def create_judge_code(prefix, pid, file_ex, size=100):
+    file_name = "%s%s%s%s" % (TEST_PATH, prefix, pid, file_ex)
+    try:
+        with open(file_name, 'w') as fp:
+            fp.write(random_word(size))
+    except (IOError, OSError):
+        print "Failed to create judge code for testing..."
+    return file_name
+
+def create_testcase_files(file_name, size=100, uploaded=False):
+    if file_name == "":
+        file_name = random_word(50)
+    if not uploaded:
+        path = TEST_PATH
+    else:
+        path = TESTCASE_PATH
+    input_file_name = "%s%s.in" % (path, file_name)
+    output_file_name = "%s%s.out" % (path, file_name)
+    try:
+        with open(input_file_name, 'w') as t_in:
+            t_in.write(random_word(size))
+        with open(output_file_name, 'w') as t_out:
+            t_out.write(random_word(size))
+    except (IOError, OSError):
+        print "Failed to create testcase files for testing..."
+    return file_name
+
+def create_testcase(problem, time_limit=1, memory_limit=32, local_files=True, uploaded_files=False):
+    testcase = Testcase.objects.create(problem=problem, time_limit=time_limit,
+                                       memory_limit=memory_limit)
+    if local_files:
+        create_testcase_files(testcase.pk)
+    if uploaded_files:
+        create_testcase_files(testcase.pk, uploaded=True)
+    return testcase
+
+def remove_file_if_exists(file_name):
+    try:
+        if os.path.isfile(file_name):
```

Comment:
Because `os.path.exists()` returns `true` when argument is a directory or a file.
Here I create a function which is called `remove_file_if_exists`, so I would like to use `os.path.isfile()` instead of `os.path.exists()`.